### ISSUE TITLE
Not an issue! More like a "thank you" word!

### ISSUE THREAD

**Issue by loneicewolf** [Jerhynh/Invoke-BSOD#1]
Hello!
Thanks for making this Repo; even if it was 2 years ago - I find these kind of things (blue screen of death's) interesting!  :)

Regards,
Will

**Comment by ItsMira02** [comment]
Of course! I was looking into undocumented NTDLL methods to bug check systems a while back and came across the NTRaiseHardError method which doesn't even require special permissions like administrator so I did some further research to figure out what sorts of other values I could pipe into it to manipulate the BSOD error message. All around it is a very interesting piece of code that works in most if not all versions of windows that utilize the NT kernel, which leads me to believe it is only intended for internal code to use when errors occur.

I appreciate you checking out the repo, I figured someone would find the information useful someday.
Cheers,
Huey
